### PR TITLE
T4d contact

### PR DESCRIPTION
**Pull request by BuzatuCos**
contact

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -14,77 +13,285 @@ nav {
...
-     list-style: none;
+	 text-decoration: none;
 	 margin-right: 90px;
 	 font-size: 1.5rem;
- }
- 
- nav ul {
+} 
+nav ul {
 	 overflow:auto;
```
Comment (jdselby24) [inline styles.css]: does nothing

File: styles.css [hunk 2]
```diff
@@ -14,77 +13,285 @@ nav {
...
 section {
 	overflow:auto;
-	height:730px;
 	width:80%;
 	margin: 0 auto;
 }
-
-.h1About{
+.titleAbout{
 	font-family: 'Lobster', 'cursive';
```
Comment (jdselby24) [inline styles.css:64]: dont need to set font again as you have already set this font for 'a' tags

File: index.html [hunk 3]
```diff
@@ -4,31 +4,60 @@
...
+	 <main>
+		 <section>
+	 	<h1 class="titleAbout"><a name="anchor2"></a>About Me</h1>
 		<div class="myImage"></div>
 		<div class="aboutPar">
 		<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
 	</div>
-	 </section>
+    </section>	 
+	 <article>
```
Comment (jdselby24) [inline index.html:35]: this should be a section not a article

File: index.html [hunk 4]
```diff
@@ -4,31 +4,60 @@
...
+		 	<div class="portBox"></div>
+		 	<div class="portBox"></div>
+		 	<div class="portBox"></div>
+		 	<div class="portBox"></div>
+		</div> 
+	</section>
+	 </article>
+	 </main>
+	 <footer>
+		 <section>
```
Comment (jdselby24) [inline index.html:54]: you don't need another section here, all styling can be done by first referencing the parent

File: styles.css [hunk 5]
```diff
@@ -14,77 +13,285 @@ nav {
...
+
+@media screen and (max-width: 750px) {
+.titleAbout{
+	padding:40px 60px;
+}
+header{
+	height:400px;
+}
+header h1{
+		font-size:3rem;
```
Comment (jdselby24) [inline styles.css]: indenting is weird here

File: styles.css [hunk 6]
```diff
@@ -14,77 +13,285 @@ nav {
...
+		padding-top:150px;
+}
+.portBox{
+		width:300px;
```
Comment (jdselby24) [inline styles.css]: and here

File: styles.css [hunk 7]
```diff
@@ -14,77 +13,285 @@ nav {
...
+	width:40px;	
+}
+.contactEmail{
+	font-size:1.5rem;	
+}
+}
+
+@media screen and (max-width: 480px){
+header h1 {
+		font-size:2rem;
```
Comment (jdselby24) [inline styles.css]: indenting is weird here

File: styles.css [hunk 8]
```diff
@@ -14,77 +13,285 @@ nav {
...
+}
+.aboutPar{
+	font-size:0.8rem;
+	margin:30px 0px 170px 0px;
+}
+.gitLogo img {
+	width:40px;	
+}
+.contactEmail{
+	font-size:1.5rem;
```
Comment (jdselby24) [inline styles.css:251]: indenting is weird here

File: styles.css [hunk 9]
```diff
@@ -14,77 +13,285 @@ nav {
...
 }
+.aboutPar{
+	font-size:0.6rem;
+	margin:20px 0px 80px 0px;
+}
+article h1{
+	font-size:2rem;
+}
+.portBox{
+		width:180px;
```
Comment (jdselby24) [inline styles.css]: indenting is weird here

**Review by jdselby24 — CHANGES_REQUESTED** [review]

File: styles.css (hunk 2 above)
Comment (BuzatuCos) [reply styles.css:64]: what you mean? to choose another font?

File: index.html (hunk 4 above)
Comment (BuzatuCos) [reply index.html:54]: i use section to have a parent that always position my elements in the center within the same distance from left to right

File: styles.css (hunk 1 above)
Comment (BuzatuCos) [reply styles.css]: fix

**Pull request merged**